### ISSUE TITLE
Obstacles deleted by geometry::helper::correct

### ISSUE THREAD

**Issue by TKoch7** [JuPedSim/jpscore#920]
**JuPedSim - JPScore**

- Current date   : 10/01/2022
- Version        : 0.8.4
- Compiler       :
- Commit hash    : id v0.8.4-945-g869817
- Commit date    : Wed Apr 15 16:39:00 2020
- Python         :  3.9.7

**Describe the bug**
Obstacles are deleted previous to the simulation (Log says "geometry::helper::correct" is called to fix geometry) and a corrected geometry file without obstacles is given as an output connected to the trajectory file. Thus, the obstacles are not shown in JPSvis (see Screenshot). However, my tests show that the obstacles are considered within the simulation and pedestrians are moving around the obstacles successfully. The JPSeditor draws the original geometry file correctly including the obstacles (see Screenshot). If the corrected geometry file is replaced by the original one, the obstacles are shown. Therefore, the problem cannot be related to JPSvis.

**Expected behavior**
Obstacles should be visible as they are in the JPScore example big_room.

**Screenshots**
![jpsvis](https://user-images.githubusercontent.com/97431830/148732010-7eb95483-3cb8-49f7-a856-a939fbb618d7.PNG)
![jpseditor](https://user-images.githubusercontent.com/97431830/148732014-bf00461e-93b5-4559-a4d8-519f262dfcd6.PNG)

**Additional context**

**Comment by chraibi** [comment]
can you share the files?

**Comment by TKoch7** [comment]
Thanks for the fast reply!

Attached you find the original files, the log and the resulting/corrected files.
[test_supermarket.zip](https://github.com/JuPedSim/jpscore/files/7839572/test_supermarket.zip)

**Comment by Ozaq** [comment]
@TKoch7 Geometry correction has been removed on the development branch and obstacles are no longer removed. The geometry file should now be copied to `results` verbatim. 

As a workaround you could copy your input geometry file over the one written by jpscore in the results folder if you do not want to switch to a development version. However when running your sample I saw that your geometry needs to be fixed. I.e. `jpscore` complains about polygons not closed, so you would need to correct this first before you can run a newer development version.

**Comment by TKoch7** [comment]
@Ozaq Thanks for the hint regarding the development version! I actually did the workaroung beforehand and it worked, though I got several error messages, when opening JPSvis. Which version did you use to run my sample? I fixed all errors I got using 0.8.4. Which version do you recommend to use?

**Comment by Ozaq** [comment]
@TKoch7 Just a few hours ago I tagged a `v0.9.0`, see https://github.com/JuPedSim/jpscore/releases/tag/v0.9.0 It is labelled as interim release because we are still in the progress to remove overcomplicated and error prone functionality. But is should be good enough (TM) to give it a try. Although it might be a lot slower because we removed multithreading (at least for now). So make sure to compile in Release mode.
